script/Ciel/Day 10/10_01_CIEL10_4_2.txt

```diff
@@ -52,7 +52,7 @@ I see... I feel like I'm getting the hang of how to put Senpai in a good mood.
 -- 「いいじゃないですか。必殺技、カッコイイ。あれでこそシエル先輩です。見惚れる事この上ない。
 "C'mon, what's wrong with calling it an ultimate move?
  It's cool.
- It fits you perfectly, too――I can't imagine anyone more captivating.
+ It fits you perfectly, too――I can't imagine anything more captivating.
```

Comment:
Ughh this one's kinda hard to make good. In verbose terms, I think the idea is
> It's just the sort of thing that I would expect you to be able to pull off, Ciel-senpai.
> After all, [it was impressive/captivating and] you're the most [impressive/captivating] person there is.

Colloquial'd up maybe something like this?
> And I mean, it fits your style, right?
>  Nobody else could turn heads quite like that.

Anyone else thoughts?

Reply:
Your version sounds good to me

Reply:
I see your point. I assumed the captivating here was describing the ultimate move and not Ciel herself hence the change.